src/20_Components/amp-next-page.html

```diff
@@ -0,0 +1,147 @@
+  </p>
+  <p class="previewOnly m2">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis venenatis, metus a tristique aliquet, enim nibh efficitur sem, ut iaculis urna justo eu diam. Nullam cursus sapien et sodales posuere. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Suspendisse potenti. Donec vitae ornare risus. Maecenas eleifend ante vel dui laoreet, et porttitor libero rutrum. Nam arcu mi, ullamcorper at risus et, pulvinar ultrices erat. In pellentesque sem vel purus auctor, ut venenatis tellus tristique. Phasellus molestie diam orci, nec gravida turpis bibendum ut. Sed sagittis aliquet lorem sed dictum.
+  </p>
+  <p class="previewOnly m2">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis venenatis, metus a tristique aliquet, enim nibh efficitur sem, ut iaculis urna justo eu diam. Nullam cursus sapien et sodales posuere. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Suspendisse potenti. Donec vitae ornare risus. Maecenas eleifend ante vel dui laoreet, et porttitor libero rutrum. Nam arcu mi, ullamcorper at risus et, pulvinar ultrices erat. In pellentesque sem vel purus auctor, ut venenatis tellus tristique. Phasellus molestie diam orci, nec gravida turpis bibendum ut. Sed sagittis aliquet lorem sed dictum.
+  </p>
+
+  <!-- ## Basic Usage -->
+  <!--
+    The `amp-next-page` component can be placed anywhere in the document
+    `body`. Subsequent pages will be added to the document as children of the
+    component, so it should usually be placed at the end of your content,
+    before any footers.
+
+    All page URLs specified in the component config must be served from the
+    same origin (protocol, domain and port) as the parent document. Canonical
+    URLs will automatically be rewritten to cache URLs when served from the
+    cache.
+  -->
+  <amp-next-page>
+    <script type="application/json">
+      {
+        "pages": [
+          {
+            "title": "News article",
+            "image": "<%host%>/img/social.png",
+	    "ampUrl": "<%host%>samples_templates/news_article/preview/"
+          }
+        ]
+      }
+    </script>
+  </amp-next-page>
+
+  <!-- ## Styling the page separator -->
+  <!--
+    By default a line will be displayed between pages to act as a separator.
+    The style of this separator can be adjusted by styling the
+    `amp-next-page-default-separator` class.
+  -->
+  <style amp-custom>
```

Comment:
This will make the page invalid, instead move this into a code snippet:

```
<!--
    By default a line will be displayed between pages to act as a separator.
    The style of this separator can be adjusted by styling the
    `amp-next-page-default-separator` class.
   
    ```css
    .amp-next-page-default-separator {
      border-bottom 2px solid red;
    }
     ```
  -->

```

Reply:
Done.